packages/frontend-v2/pages/home.tsx

```diff
@@ -87,7 +87,10 @@ const HomePage: NextPage = () => {
   useEffect(() => {
     // once the student is fetched, set the selected plan id to the primary plan id
     if (student && selectedPlanId === undefined) {
-      setSelectedPlanId(student.primaryPlanId);
+      if(student.plans.length > 0){
```

Comment:
super nit: try to get prettier to run before save so that whenever you make changes its always properly formatted

Reply:
+1